src/index.js

```diff
@@ -9,4 +9,13 @@ if (!token) {
 
 client.on("voiceStateUpdate", require("./VoiceUpdate"));
 
+client.on('message', async (msg) => {
```

Comment:
I think we should put this in its own file.

```suggestion
client.on('message', require("./HelpMessage"));
```